package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Looks good!  One minor issue, you seemed to forget a few devDependencies, see comment below.